Sabbath School/Settings/Controller/QuarterlySettingsController.swift

```diff
@@ -37,8 +37,12 @@ class QuarterlySettingsController: ASViewController<ASDisplayNode> {
         tableNode?.dataSource = self
         
         title = "Quarterly".uppercased().localized()
-        
-        titles = [["Adult".localized(), "Collegiate".localized()]]
+       
+        if currentLanguage().code == "en" {
+            titles = [["Adult".localized(), "Collegiate".localized()]]
+        } else {
+            titles = [["Adult".localized()]]
```

Comment:
This might be a temporally solution. If the user isn't using English then they can only select the 'adult' edition which would behave as previous versions of the app did. If they are using English then they can choose the different editions :-)